qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -444,10 +444,308 @@ def particle_number(orbitals, mapping="jordan_wigner", wires=None):
     return observable([table], mapping=mapping, wires=wires)
 
 
+def one_particle(t_matrix_elements, core=None, active=None, cutoff=1.0e-12):
+    r"""Generates the table of matrix elements of a given one-particle operator
+    required to build many-body qubit observables.
+
+    Second quantized one-particle operators are expanded in the basis of single-particle
+    states as
+
+    .. math::
+
+        \hat{T} = \sum_{\alpha, \beta} \langle \alpha \vert \hat{t} \vert \beta \rangle
+        [\hat{c}_{\alpha\uparrow}^\dagger \hat{c}_{\beta\uparrow} +
+        \hat{c}_{\alpha\downarrow}^\dagger \hat{c}_{\beta\downarrow}].
+
+    In the equation above the indices :math:`\alpha, \beta` run over the basis of spatial
+    orbitals :math:`\vert \alpha \rangle = \phi_\alpha(r)`. Since the operator :math:`t`
+    acts only on the spatial coordinates, the spin quantum numbers are indicated explicitly
+    with the up/down arrows. The operators :math:`\hat{c}^\dagger` and :math:`\hat{c}` are the
+    particle creation and annihilation operators, respectively, and
+    :math:`\langle \alpha \vert \hat{t} \vert \beta \rangle` denotes the matrix elements of
+    the operator :math:`\hat{t}`
+
+    .. math::
+
+        \langle \alpha \vert \hat{t} \vert \beta \rangle = \int dr ~ \phi_\alpha^*(r)
+        \hat{t}(r) \phi_\beta(r).
+
+    If an active space is defined (see :func:`~.active_space`), the summation indices
+    run over the active orbitals and the contribution due to core orbitals is computed as
+    :math:`T_\mathrm{core} = 2 \sum_{\alpha\in \mathrm{core}}
+    \langle \alpha \vert \hat{t} \vert \beta \rangle`.
+
+    Args:
+        t_matrix_elements (array[float]): 2D NumPy array with the matrix elements
+            :math:`\langle \alpha \vert \hat{t} \vert \beta \rangle`
+        core (list): indices of core orbitals, i.e., the orbitals that are
+            not correlated in the many-body wave function
+        active (list): indices of active orbitals, i.e., the orbitals used to
+            build the correlated many-body wave function
+        cutoff (float): Cutoff value for including matrix elements. The
```

Comment:
Cutoff might be changed to cutoff for consistency.

Reply:
Here I am following guidelines for PennyLane docstrings.
See here https://pennylane.readthedocs.io/en/stable/development/guide/documentation.html#punctuation-and-spelling